devel/grpc/Portfile

```diff
@@ -20,8 +20,8 @@ long_description \
     These libraries enable communication between clients and \
     servers using any combination of the supported languages.
 
-checksums       rmd160  39d04ec6d0d5440c115454a41c21240d7d9f20b0 \
-                sha256  2f1420101d43fcaa3c2a9bca1fd4c6826af490ef703bb12b4196d9f255faef14
+checksums       rmd160  63cf7566f66feba0b2fc443503ab29329ee5c202 \
+                sha256  02f8b368a20b67f4c57727df45d7969fb6a3ba0a1246a7c843c3a4c8e4946a64
```

Comment:
FYI, these days we often add a "size" parameter.

Reply:
Alright is this documented? I don't see it mentioned in man portfile.

Reply:
`size` is just the file size in bytes. Usually you can generate this with `sudo port -v checksum`, which outputs the complete checksums option for the default types on a checksum mismatch.